pcdsdevices/pseudopos.py

```diff
@@ -31,7 +31,10 @@ class PseudoSingleInterface(FltMvInterface, PseudoSingle):
         notepad_metadata={'record': 'ai', 'default_value': 0.0},
     )
 
-    def __init__(self, prefix='', parent=None, **kwargs):
+    # Current Dial position
+    dial_position = Cpt(EpicsSignalRO, '.DRBV', kind='normal')
```

Comment:
I'm not positive that this one exists either

Reply:
yeah, same as above probably - thank you

Reply:
I can try using this but not sure if works for all caseses:
```
# Current Dial position
dial_position = FCpt(EpicsSignalRO, '{self.parent.prefix}.DRBV', kind='normal')
```

Reply:
Does this work for any cases?

Reply:
Works for this guy: xcs_lxt_fast.delay - but i am not sure if this is a good device to test against

Reply:
I don't think this is correct for that example- the value you're getting is in `mm`

Reply:
I see that it is the correct pre-calculation value- but I don't think we can assume that this is true in all cases

Reply:
Ohh gosh, my brain stops working on Fridays I think - sorry, this line should have been taken out, and only this one left:
```
# Current Dial position
    dial_position = FCpt(EpicsSignalRO, '{self.parent.prefix}.DRBV',
                         kind='normal')
```

Reply:
but regardless, i need to try your suggestions below